taskcluster/kinds/extract-best/kind.yml

```
@@ -81,11 +82,13 @@ tasks:
                 - >-
                     zstd -d --rm $MOZ_FETCHES_DIR/*.zst &&
                     export PYTHONPATH=$PYTHONPATH:$VCS_PATH &&
+                    pip install --upgrade pip &&
+                    pip install -r $VCS_PATH/pipeline/translate/requirements/bestbleu.txt &&
                     python3 $VCS_PATH/pipeline/translate/bestbleu.py
                     -i "$MOZ_FETCHES_DIR/file.{this_chunk}.nbest"
```

Suggestion: While you are touching this code, can you expand the arguments?

```suggestion
                    --nbest "$MOZ_FETCHES_DIR/file.{this_chunk}.nbest"
```